docs/proposals/distributed_architecture/docs/06-Host_Regenesis_and_PXE-Based_Provisioning.md

```diff
@@ -0,0 +1,73 @@
+## Host Regeneration and PXE-Based Provisioning
+
+### Gap Definition and Improvement Objectives
+
+Currently, Flintlock lacks automated and streamlined procedures for securely reprovisioning and reintegrating detached hosts back into the cluster. Introducing PXE-based provisioning will enable automated host regeneration and reduce manual operational tasks.
```

Comment:
> detached hosts into the cluster

I'm taking that to mean the machines that run flintlock. 

I think PXE based provisioning is out of scope of this.